Source/missiles.cpp

```diff
@@ -264,13 +267,12 @@ bool MonsterMHit(int pnum, int m, int mindam, int maxdam, int dist, int t, bool
 	if (CheckMonsterHit(monster, &ret))
 		return ret;
 
+	if (hit >= hper) {
 #ifdef _DEBUG
-	if (hit >= hper && !debug_mode_key_inverted_v && !debug_mode_dollar_sign)
-		return false;
-#else
-	if (hit >= hper)
-		return false;
+		if (!DebugGodMode)

```

Comment:
I hate this identation so much, can't we deal with it somehow? 😅

Reply:
So do I, but it's a step up from the duplicate